src/styles/utils/cycles.scss

```diff
@@ -0,0 +1,28 @@
+@each $day, $index in $days-of-week {
+  .calendar--start-day-#{$day} {
+    .calendar__day:first-child {
+      margin-left: calc(#{$cell-size} * #{$index} + $gap * #{$index});
+    }
+  }
+}
+
+
+@for $days from 28 through 31 {
+  .calendar--month-length-#{$days} {
+    @for $i from ($days + 1) through 32 {
+      .calendar__day:nth-child(#{$i}) {
+        display: none;
+      }
+    }
+  }
+}
+
+
+
+@for $i from 1 through $default-month-length {
+  .calendar__day:nth-child(#{$i})::before {
+    content: '#{$i}';
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+  }
```

Comment:
Font family and size should be defined in a separate placeholder selector for reusability and to avoid duplication, as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).